### DISCUSSION TITLE
Timeless Sydney v1.8.2

### DISCUSSION THREAD

**Discussion by xbzstudio** [xbzstudio/Timeless-Sydney#88] [公告]
修复了一些bug

<hr /><em>This discussion was created from the release <a href='https://github.com/xbzstudio/Timeless-Sydney/releases/tag/TimlessSydney1.8.2'>Timeless Sydney v1.8.2</a>.</em>

**Comment by kigg1234** [comment]
請問要如何建置在ubnntu 20.4 上?

**Comment by Andy-lly** [comment]
一直连接失败是什么原因呢？

**Comment by nuclisland** [comment]
AttributeError: module 'attr' has no attribute 'frozen'
